source/CMakeLists.txt

```diff
@@ -250,10 +250,22 @@ target_include_directories(appleii PRIVATE
   )
 
 # this one appears in header files
+if ("${CMAKE_SYSTEM_NAME}" STREQUAL "FreeBSD")
+# FreeBSD could already have "zip.h" (from libzip) in /usr/local/include
+# which takes precedence than the one in /usr/local/include/minizip.
+# "zip.h" from libzip is not what we want here.
+# The following changes to use include path /usr/local/include for minizip
+# so that the code can include "minizip/zip.h" explicitly.
+target_include_directories(appleii PUBLIC
+  ${CMAKE_CURRENT_BINARY_DIR}  # for config.h
+  ${MINIZIP_INCLUDE_DIRS}/..
```

Comment:
I see the issue, but it is not limited to FreeBSD. What happens if I install https://packages.ubuntu.com/noble/libzip-dev ?
There must be a more general solution

Reply:
I have tried and there are no issues. The reason must be
1) if I want minizip, then I add `-I/usr/include/minizip' and although `zip.h` is ambiguous, this explicit `-I` wins over the default one (used by `libzip`)
2) If I want zip, no problem as I do not add the `minizip` `-I`

So, the issue is probably related to a user who have both `libzip` and `minizip` in `/usr/local` which breaks the (weak) order of includes.

Reply:
OK, now I see better. In FreeBSD they are all installed in `/usr/local` so they cannot coexist.
Which is really odd, maybe nobody is using it, someone should file a bug.

The fact is that the command `pkg-config minizip -cflags` in FreeBSD returns `-I/usr/local/include/minizip'.

Reply:
I think it is exactly the order of `-I` problem. `pkg-config` is returning the right path for `minizip`. But some other libs is likely returning the common `/usr/local/include` and happens to be earlier in the include path. The kind of problem does happen to FreeBSD ports occasionally too. Ideally for FreeBSD `/usr/local/include` should always be the last path. But that's very difficult to control. Neither is to have `/usr/local/include/minizip` always in front, just not obvious to me how within CMake that can be done reliably. If all Linux distros always put `minizip` headers in its own subdirectory, the hack here could work universally and so in AppleWin code we could simply `#include <minizip/zip.h>`.